README.md

```diff
@@ -1,22 +1,30 @@
 # Regiodesics
 
-# Tools
+Tools to generate relative distance and orientation fields in 3D brain atlases.
+
+# Apps
 
 ## layer\_segmenter
 
-Multi-purpose application that can be used to generate top and bottom shells of a brain region.
+![Screenshot of layer_segmenter application.](https://github.com/seirios/regiodesics/assets/7497849/c60e2adb-0e7b-4664-85bb-70d0978cc99d)
```

Comment:
This should reference the Bluebrain repo once the PR is ready.

Reply:
Fixed :)